components/button/button-subtle.js

```diff
@@ -125,20 +122,21 @@ class ButtonSubtle extends ButtonMixin(RtlMixin(LitElement)) {
 				:host([active]:not([disabled])) button .d2l-button-subtle-content {
 					color: var(--d2l-color-celestine-minus-1);
 				}
-				:host([icon]) .d2l-button-subtle-content,
-				:host([dir="rtl"][icon][icon-right]) .d2l-button-subtle-content {
-					padding-left: 1.2rem;
-					padding-right: 0;
+				.d2l-button-subtle-has-icon .d2l-button-subtle-content {
+					padding-inline: 1.2rem 0;
 				}
-				:host([dir="rtl"][icon]) .d2l-button-subtle-content,
-				:host([icon][icon-right]) .d2l-button-subtle-content {
-					padding-left: 0;
-					padding-right: 1.2rem;
+				:host([icon-right]) .d2l-button-subtle-has-icon .d2l-button-subtle-content {
+					padding-inline: 0 1.2rem;
 				}
 
-				d2l-icon.d2l-button-subtle-icon {
-					color: var(--d2l-color-celestine);
+				d2l-icon.d2l-button-subtle-icon,
```

Comment:
I _think_ applying `display: none` to the `d2l-icon` isn't necessary since it is and was conditionally rendered based on the precense of the `icon` property. Further down it then gets `display: inline-block` which could have just been applied from the start?